picoscope/ps4000a.py

```diff
@@ -342,6 +388,35 @@ def _lowLevelGetValues(self, numSamples, startIndex, downSampleRatio,
         self.checkResult(m)
         return (numSamplesReturned.value, overflow.value)
 
+    def _lowLevelSetDeviceResolution(self, resolution):
+        self.resolution = resolution
+        m = self.lib.ps4000aSetDeviceResolution(
+            c_int16(self.handle),
+            c_enum(resolution))
+        self.checkResult(m)
+
+    def _lowLevelGetValuesBulk(self, numSamples, fromSegment, toSegment,
+                               downSampleRatio, downSampleMode, overflow):
+        """Copy data from several memory segments at once."""
+        overflowPoint = overflow.ctypes.data_as(POINTER(c_int16))
+        m = self.lib.ps4000aGetValuesBulk(
```

Comment:
did you change anything but the formatting here? Is it possible for you to undo this change without too much work?


it is REALLY difficult to check the correctness of this kind of code, so I'm much more in favour of keeping ugly formatting than changing it.

Reply:
The ps4000a looked like it was a copy of ps4000 with a find and replace to refer to the ps4000a library. The low level call was modified to be more in line with the ps5000a library (uint16's modified to uint32's etc).

Reply:
got it, thanks! keep it as is.